service/history/queue/action.go

```diff
@@ -50,13 +56,33 @@ type (
 	GetStateActionResult struct {
 		States []ProcessingQueueState
 	}
+
+	// GetTasksAttributes contains the parameter to get tasks
+	GetTasksAttributes struct{}
+	// GetTasksResult is the result for performing GetTasks Action
+	GetTasksResult struct {
+		tasks []task.Task
```

Comment:
Maybe update the type to be `[]*types.CrossClusterTaskRequest`? There's not much the caller can do with the Task interface and construct a proper TaskRequest I guess?

Reply:
updated